packages/whatsapp_bot_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: whatsapp_bot_flutter
 description: Whatsapp bot using puppeteer and whatsapp web scraping with flutter desktop/web and dart projects support
-version: 2.0.0
+version: 2.0.1
```

Comment:
@loic-hamdi please do not change versions, we can wait for few more features addition or fixes and i will do it in final update